Chap_API_Proc_Mgmt.tex

```diff
@@ -806,21 +851,19 @@ \subsection{\code{PMIx_Connect}}
 \reqattrend
 
 \optattrstart
-The following attributes are optional for \ac{PMIx} implementations:
-
-\pasteAttributeItem{PMIX_ALL_CLONES_PARTICIPATE}
-
-
 The following attributes are optional for host environments that support this operation:
 
```

Comment:
confirmed that the optattrstart only contains the banner and not the intro text.  Something to consider since we repeat that "The following attributes are optional...." everywhere in the standard, but that is an issue for another PR.